test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -9481,7 +9483,42 @@ def create_import_hook(self, name, attrs=None, body=None, overwrite=True):
 
         # In 12.0 A MoM hook must be enabled and the event set prior to
         # importing, otherwise the MoM does not get the hook content
-        return self.import_hook(name, body)
+        ret = self.import_hook(name, body)
+
+        # In case of mom hooks, make sure that the hook related files
+        # are successfully copied to the MoM
+        try:
+            if attrs['event'].startswith('exec'):
```

Comment:
This won't work if there are multiple events.
Docs say that if you are adding multiple events, you have to add quotes and separate with commas: `“<event name>,<event name>”`
You can see this in the cgroups hook: https://github.com/PBSPro/pbspro/blob/bd75e3a186128014e3051374fd31e5ffd70577a0/test/tests/functional/pbs_cgroups_hook.py#L661-L667